python/run_benchmark.sh

```diff
@@ -362,17 +363,20 @@ fi
 
 # Logistic Regression Classification
 if [[ "${MODE}" =~ "logistic_regression" ]] || [[ "${MODE}" == "all" ]]; then
-    if [[ ! -d ${gen_data_root}/classification/r${num_rows}_c${num_cols}_float32.parquet ]]; then
-    # same data set as random forest classifier
+
+    data_path=${gen_data_root}/classification/r${num_rows}_c${num_cols}_float32_ncls${num_classes}.parquet
```

Comment:
So with this we have just num_classes > 2?   We should retain a run with num_classes=2.

Reply:
Can we retain a run with num_classes=2 in hte benchmark script (that's an important special case).

Reply:
Yeah, just revised the PR to run both 2 and 10 classes for LR default.